GG/GG/Font.h

```diff
@@ -147,6 +147,9 @@ class GG_API Font
             pair.second. */
         Substring(const std::string& str_, const IterPair& pair);
 
+        /** Rebind to a different string.*/
```

Comment:
I expanded the documentation of these functions where they were defined in #1039.